ccmlib/scylla_repository.py

```diff
@@ -255,7 +263,7 @@ def setup(version, verbose=True, skip_downloads=False):
     if type_n_version[0] == 'release':
         version = normalize_scylla_version(version)
         type_n_version = version.split(os.path.sep, 1)
-    version_dir = version_directory(version) if not skip_downloads else None
+    version_dir = version_directory(version + "-debug" * scylla_debug) if not skip_downloads else None
```

Comment:
this uses the fact that bool is just an 1 or 0, and a string times 0 is the empty string

Reply:
That is very clever, but the fact that you felt that it deserved a comment means it might be not verbose enough to other users, especially in the future. Would simplifying it so that it is more readable keep it as compact as it is now or would it be much longer?

Reply:
I agree.  Something like `f"{version}{'-debug' if scylla_debug else ''}" would even be more readable IMO